modules/earthengine-layers/src/earth-engine-layer.js

```diff
@@ -80,7 +80,15 @@ export default class EarthEngineLayer extends CompositeLayer {
     }
 
     // Evaluate map
-    const map = await eeObject.getMapAsync(props.visParams);
+    const map = await new Promise((resolve, reject) =>
+      eeObject.getMap(props.visParams, (value, error) => {
```

Comment:
There was supposedly a `promisify` function in the ee api, maybe see if we can find it?

Reply:
Also maybe put this in a helper function/method?

Reply:
There's no public promise function in the Earth Engine API. It's possible that there are private promise functions that aren't currently exported.

I was looking into a reusable `promisify` function. Is it possible to do something like this?
```js
export function promisify(fn, ...args) {
  return new Promise((resolve, reject) =>
    fn(...args, (value, error) => {
      if (error) {
        reject(error);
        return;
      }
      resolve(value);
    })
  );
}
const map = await promisify(eeObject.getMap, props.visParams);
```
It didn't work, and I was thinking maybe `this` was pointing to the wrong object.